src/scripts/notebooks/clinvar-udn-submission-analysis/data/output/clinvar_udn_submissions-no_mondo-disease_name_only.txt

```diff
@@ -0,0 +1,145 @@
+Name
+1p13.3 deletion syndrome
+ABCA2-related disorder
+ACOX1-related disorder
+ADCY2-related disorder
+ADGRV1-related myoclonic epilepsy
+AGTPBP1-related disorder
+AIFM1-related hypomyelination with spondylometaphyseal dysplasia
+ARX-associated condition
+ATAD3 gene cluster related condition
+ATP5F1A-related disorder
+ATP5PO-related disorder
+AXIN2-related disorder
+Adult hypophosphatasia
+BAP1-associated neurodevelopmental disorder
+BICRA-related disorder
+BLK-related disorder
+Bone mineral density quantitative trait locus 18
+CACNA1A-related disorder
+CACNA1C-related disorder
+CDH2-related disorder
+CDK19-related disorder
+CDKL5-related disorder
+CHRNA3-related disorder
+COPB2-related disorder
+CRELD1-related disorder
+CTNNA1-associated FEVR
+Cerebellar atrophy
+Congenital titinopathy
+DAGLA-related disorder
+DHX9-related disorder
+DNM1L-related movement disorder
+DOHH-related disorder
+DONSON-related Meier-Gorlin syndrome
+DROSHA-related neurodevelopmental disorder
+DSP-related arrhythmogenic cardiomyopathy
+Decreased activity of mitochondrial ATP synthase complex
+Diarrhea

```

Comment:
Are these all the new terms that will be added to Mondo? @twhetzel did you get synonyms to see if any of these terms existed in Mondo already from other sources besides medgen? ClinGen has requested gene-related diseases like these, so some of these terms could _potentially_ exist in Mondo that came from them (I spot checked a few but didn't see any examples).

But diarrhea is a synonym for MONDO:0001673 diarrheal disease

Reply:
Yeah, there will be some false positives in here. My aim was to filter out the known/easily found positives knowing the rest would need curation and we are familiar with our Mondo class naming policy and DOSDP patterns so can sort out this more easily. 

The choice of file I eventually used is in this [comment](https://github.com/monarch-initiative/mondo/issues/7773#issuecomment-2161452894). But again, this was intended as a quick check to filter out the easy things to help scope out the curation effort.